README.md

```diff
@@ -35,3 +39,156 @@ understand.
+
+It will also leverage the Torque integration of the MPI library being used by default, by launching MPI processes using `pbsdsh` rather than `ssh`.
+
+
+# Configuring `mympirun`
+
+* `mympirun` command line options
+* `$MYMPIRUN_*` environment variables
+* configuration files (see `--configfiles`)
+
+# Controlling number of processes
+
+## `--hybrid` / `-h`
+The `hybrid` or `h` option requires one integer. This integer will be the number of processes started on each available physical node.
+        
+    $ echo $PBS_NUM_NODES
+    2
+
+    $ mympirun --hybrid 2 hostname
+    node2157.delcatty.os
+    node2157.delcatty.os
+    node2158.delcatty.os
+    node2158.delcatty.os
+
+## `--universe`
+The `universe`option also requires one integer. This integer will be the exact number of processes started by mympirun (independent of the number of nodes). When using `universe` for multiple processes, mympirun will try to balance the processes on the available nodes.
+        
+    $ echo $PBS_NUM_NODES
+    2
+
+    $ mympirun --universe 1 hostname
+    node2157.delcatty.os
+    
+
+## `--double` and `--multi`
+As the name suggests, when using the `double` option,  mympirun will start double the amount of processes as requested. The `multi`option works the same but it requires an integer, indicating the multiplier, for example, `multi 3` will start triple the amount of processes. This means `double` and `multi 2` will have the exact same effect.
+
+    $ echo $PBS_NUM_NODES
+    2
+
+    $ echo $PBS_NUM_PPN
+    4
+
+    $ mympirun --double hostname
+    node2157.delcatty.os
+    node2157.delcatty.os
+    node2157.delcatty.os
+    node2157.delcatty.os
+    node2157.delcatty.os
+    node2157.delcatty.os
+    node2157.delcatty.os
+    node2157.delcatty.os
+    node2158.delcatty.os
+    node2158.delcatty.os
+    node2158.delcatty.os
+    node2158.delcatty.os
+    node2158.delcatty.os
+    node2158.delcatty.os
+    node2158.delcatty.os
+    node2158.delcatty.os
+    
+
+
+# Controlling output
+
+Use `--output` to redirect your mympirun output to a file instead of stdout/stderr. 
+
+    $ mympirun --output out.txt hostname
+    
+    $ cat out.txt
+    node2157.delcatty.os
+    node2157.delcatty.os
+    node2157.delcatty.os
+    node2157.delcatty.os
+    node2158.delcatty.os
+    node2158.delcatty.os
+    node2158.delcatty.os
+    
+
+
+# Passing down environment variables
+
+The environment variables that mympirun passes to the MPI environment are `LD_LIBRARY_PATH`, `PATH`, `PYTHONPATH`, `CLASSPATH`, `LD_PRELOAD`, `PYTHONUNBUFFERED` and all environment variables that are prefixed with `OMP`, `MKL`, `KMP`, `DAPL`, `PSM`, `IPATH`, `TMI`, `PSC`, `O64`, and `VSMP`
+
+To add other variables to this list, you can use the mympirun option  `--variablesprefix`. This option requires a comma-seperated list of strings that are either the exact variable name or a prefix that matches the environment variable(s).
```

Comment:
clarify difference between full environment variable names and prefixes (`_*` is appended automagically)